server/src/domain/user/validation.service.impl.spec.ts

```diff
@@ -1,19 +1,13 @@
 import {Test, TestingModule} from '@nestjs/testing';
 import {ValidationServiceImpl} from './validation.service.impl';
 import {instance, mock, when} from 'ts-mockito';
-import {BridgeService} from './bridge.service';
 import {Web3Module} from '../../port/adapter/service/blockchain/ethereum/web3/web3.module';
 import {ConfigModule} from 'nestjs-config';
 import * as path from 'path';
 import { Web3BridgeService } from '../../port/adapter/service/blockchain/ethereum/web3/web3.bridge.service';
 
 describe('ValidationServiceImpl', () => {
 
-    class BrideServiceImpl implements BridgeService {
-        recover(message: string, signature: string): string {
-            return '';
-        }
-    }
     const mockBridgeService = mock(Web3BridgeService);
```

Comment:
i think Web3BridgeService should not be imported to domain layer. 

i think BridgeServiceImpl should be replace Web3BrdigeService.

Reply:
@inDlife I've already done pull rebase but cannot find BridgeServiceImpl :( Would you tell me where it is?

Reply:
Sure im gonna comment it when i go home

Reply:
```
class BrideServiceImpl implements BridgeService {
        recover(message: string, signature: string): string {
            return '';
        }
    }
    const mockBridgeService = mock(BrideServiceImpl);
```

Reply:
@inDlife Ahhh there was typo! it's 'Bridge'ServiceImpl, not 'Bride'ServiceImpl. I will fix it!